_posts/2019-10-08-lincolnhack.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "LincolnHack is the greatest hack in the East Midlands!"
+date:   2019-10-08 20:17:00 +0000
+author: rob_willie
+categories: community contributions
+---
+
+<p style="text-align: center">
+<img style="width: 260px;" src='/images/Vector-Imp.svg'/>
+</p>
+
+# <center>LincolnHack - Pies not Pizza</center>
+
+The first LincolnHack was held in 2011. It was a very small affair with an operational budget of £500. It was a lot of work to run, mistakes were made, and wasn’t as good as LeedsHack anyway, so we parked our aspirations in favor of attending our favourite hack event.
+
+In 2016, LeedsHack faltered when its main organiser, [the Hodge](https://twitter.com/thehodge), retired from running the event. Concerned that our favourite hackathon was no more, we decided to reboot LincolnHack in LeedsHack’s image while adding some excellent food choices along the way. The most important thing we wanted to achieve was to make LincolnHack about building relationships, not organising a tedious networking event trading business cards with people whom you’ll never see again.
+The new LincolnHack was everything we set out for it to be. People met and magic happened: startups were born, employment opportunities revealed themselves, friends were made, knowledge exchanged, and unusual and interesting things created. From this, a community was born. 
+
+LincolnHack has been running for 4 years strong, and we’ve stayed true to our guiding principles of building relationships in a supporting and inclusive community. You can show up to LincolnHack not knowing anyone, join a team, and by the end of the weekend be friends for life. And the food we get for the event is incredible. Indeed, this was a conscious decision — great food is a social thing.
+
+In 2018, the Hodge came back to host our event — returning with some of that old LeedsHack magic. We tried out some new ideas (such as Crap Prizes — everyone gets a crap prize), and people left smiling.
+
+In 2019 we move to a fabulous new building, Mosaic @ Thomas Parker House. You can expect great food (PIES!), fun, and surprises. Our new venue has plenty of breakout spaces and is smack bang in the centre of Lincoln. It’s as if the space was designed for a Hackathon.
+
+LincolnHack 2019 runs on 16th and 17th November 2019, at Mosaic Lincoln. Doors open at 9am. Please see (LincolnHack.org)[https://lincolnhack.org] for more info.
```

Comment:
The link brackets are the wrong way round - very easy done!

The PR gives you a preview of the built site with a different link for each build, it's under the merge 'checks' section,

https://deploy-preview-83--webuiltthisforhacktoberfest.netlify.com/community/contributions/2019/10/08/lincolnhack.html

(FYI - this link won't stay alive if the PR is merged)